src/libraries/System.Private.CoreLib/src/System/Numerics/Matrix4x4.cs

```diff
@@ -27,55 +28,71 @@ public struct Matrix4x4 : IEquatable<Matrix4x4>
             0f, 0f, 0f, 1f
         );
 
-        /// <summary>Value at row 1, column 1 of the matrix.</summary>
+        /// <summary>The first element of the first row.</summary>
```

Comment:
In this case, I think there is probably a good in between between what we had and what is here.

Matrices are in terms of columns and rows and so I'd think "element" may be initially confusing to some users.

Reply:
As the area owner, feel free to provide a suggested text and I can apply it.

Reply:
I don't want to block this PR on it, since we aren't changing anything.

I think it might be good to see if we can get a best/recommended practices for documentation text though so we can try to:
1. Be overall consistent across .NET
2. Find something that works in cases where the terminology may end up differing

Reply:
Not sure what the action item is here. But if you'd like us to improve the docs, feel free to provide `` ```suggestion`` comments so I can bulk apply them.